js/components/controlBar.js

```diff
@@ -259,6 +259,47 @@ var ControlBar = React.createClass({
       totalTime = Utils.formatSeconds(this.props.duration);
     }
 
+
+    var volumeBars = [];
+    for (var i = 0; i < 10; i++) {
+      //create each volume tick separately
+      var turnedOn = this.props.controller.state.volumeState.volume >= (i + 1) / 10;
+      var volumeClass = ClassNames({
+        "oo-volume-bar": true,
+        "oo-on": turnedOn
+      });
+      var barStyle = null;
+      if(turnedOn === true){
+          barStyle = {backgroundColor: this.props.skinConfig.controlBar.volumeControl.color ?
+              this.props.skinConfig.controlBar.volumeControl.color : this.props.skinConfig.general.accentColor};
+      }
+      else{
+          barStyle = {backgroundColor: this.props.skinConfig.controlBar.volumeControl.inactiveColor ?
+                  this.props.skinConfig.controlBar.volumeControl.inactiveColor : this.props.skinConfig.general.inactiveAccentColor};
+      }
+      volumeBars.push(<a data-volume={(i + 1) / 10} className={volumeClass} key={i}
+        style={barStyle}
+        onClick={this.handleVolumeClick}
+        aria-hidden="true"></a>);
+    }
+
+    var volumeSlider = <div className="oo-volume-slider"><Slider value={parseFloat(this.props.controller.state.volumeState.volume)}
```

Comment:
How does this new code interact with the old code to render the volume slider? I don't see any code removed, so was curious about the matter.